roles/ceph-osd/tasks/check_gpt.yml

```diff
@@ -17,3 +17,4 @@
   when:
     - not item.0.get("skipped")
     - item.0.get("rc", 0) != 0
+    - '"Partition Table: gpt" not in item[0].stdout'
```

Comment:
I suspect this check could replace the one at line 19. It's more accurate and captures what the task wants to achieve.